app/actions/local/channel.ts

```diff
@@ -70,21 +70,22 @@ export const switchToChannel = async (serverUrl: string, channelId: string, team
         return {error};
     }
 
-    return {error: undefined};
+    return {models};
 };
 
-export const localRemoveUserFromChannel = async (serverUrl: string, channelId: string) => {
+export const localRemoveCurrentUserFromChannel = async (serverUrl: string, channelId: string, prepareRecordsOnly = false) => {
```

Comment:
I don't think we need the local prefix

Reply:
I foresee a remote action with the same name when we create the "Leave Channel" button. It will be the "removeCurrentUserFromChannel" remote action that will call the "localRemoveCurrentUserFromChannel".

Reply:
I planned to remove it, but it seems I didn't. Will fix.